src/encoded/types/experiment.py

```diff
@@ -177,6 +176,102 @@ def related_series(self, request, related_series):
     def superseded_by(self, request, superseded_by):
         return paths_filtered_by_status(request, superseded_by)
 
+    @calculated_property(schema={
+        "title": "Protein tags",
+        "description": "The protein tags introduced through the genetic modifications of biosamples investigated in the experiment.",
+        "comment": "Do not submit. This field is calculated through applied_modifications.",
+        "type": "array",
+        "notSubmittable": True,
+        "items": {
```

Comment:
If you don't return empty list below, you have `"minItems": 1,` here.

Reply:
Agreed, will add.